website/hw/hw2.rst

```diff
@@ -101,36 +94,55 @@ However, as we continue to develop libgeometry, we realize that we need
 to perform segment operations in a variety of other places. So, it now
 makes sense to refactor our code to add a Segment module.
 
+You will do
+so by performing the tasks below but, as you do, remember that you
+are just *refactoring* the code. Most notably, we are going to change
+its decomposition and organization, but not its internal logic.
+This means that you do not need to understand how
+segment intersections are actually computed (in fact, no part
+of this assignment hinges on knowing how the intersections are
+actually computed).
+
 You must do the following:
 
--  **Task 1** (30 points) Add ``segment.c`` / ``segment.h`` files with
+-  **Task 1** Add ``segment.c`` / ``segment.h`` files with
    the implementation and interface of a new Segment data type. Make
    sure to include new/init/free functions (like the ones for Point and
    Polygon). You will also need to update the Makefile to make sure this
    new file is compiled and included in the libgeometry library. Just
    update the ``SRCS`` variable in the Makefile to include your new
    ``segment.c`` file.
```

Comment:
Would we want to add note about `make clean` here?

Reply:
I have added a suggestion to use `make clean` if their code doesn't build after editing the Makefile.